docs/decisions/0019-forced-redemption.rst

```diff
@@ -0,0 +1,52 @@
+0019 Forced Redemption
+**********************
+
+Status
+======
+
+Accepted - April 2024
+
+Context
+=======
+There is frequently a need to force through a redemption (and related enrollment/fulfillment)
+of a particular learner, covered by a particular subsidy access policy, into
+some specific course run. This needs exists for reasons related to upstream
+business constraints, notably in cases where a course is included in a policy's catalog,
+but the desired course *run* is not discoverable due to the current state of its metadata.
+
+Some recent work has provided an ``allow_late_enrollment`` directive to be included
+in the ``metadata`` of the subsidy redeem payload.  This solution will utilize that directive,
+because it causes enrollment and/or external fulfillment to be forced through in
+downstream systems.
+
+Decision
+========
+* We'll expose a model via Django Admin that allows staff users to force redemption
```

Comment:
nit: Should this include the name of the proposed model `ForcedPolicyRedemption`? The updated README below mentions the creation of the new model `ForcedPolicyRedemption`, but the ADR itself doesn't explicitly mention the model name.